audit_exceptions/versioned_dependencies_conflicts_allowlist.json

```diff
@@ -1,10 +1,7 @@
 [
-  "anime-downloader",
   "chapel",
   "emscripten",
```

Comment:
`chapel` needs revision bump with migration. `emscripten` can be modified in version bump PR.